bundles/CoreBundle/Command/ThumbnailsImageCommand.php

```diff
@@ -188,7 +188,8 @@ private function fetchThumbnailConfigs(InputInterface $input): array
                         $resConfig->setHighResolution($resolution);
                         $thumbnailsToGenerate[] = $resConfig;
 
-                        if (!$input->getOption('skip-webp') && $resConfig->getFormat() === 'SOURCE') {
+                        $isWebPAutoSupport = \Pimcore::getContainer()->getParameter('pimcore.config')['assets']['image']['thumbnails']['webp_auto_support'];
```

Comment:
```suggestion

                        $config = \Pimcore\Config::getSystemConfiguration();
                        $isWebPAutoSupport = $config['assets']['image']['thumbnails']['webp_auto_support'] ?? false;
```

Reply:
@NiklasBr one last change please (sorry, forgot in my suggestion)...since we just need asset config, we can pass offset to getSystemConfiguration() i.e. `\Pimcore\Config::getSystemConfiguration('asset');`

Reply:
No worries. Didn't I do it already in d7a71bd2ca0ac20b8c86253eb28143b7bffbf9e2?